### ISSUE TITLE
Crash since upgrade to Rollbar 2.2.4

### ISSUE THREAD

**Issue by almacha** [rollbar/rollbar.js#409]
After upgrading to Rollbar 2.2.4 I run into the following error when running a script that used to work:
`/Users/raphaelchampeimont/nodejs-doctrine/node_modules/lodash/isNative.js:35
    throw new Error(CORE_ERROR_TEXT);
    ^

Error: Unsupported core-js use. Try https://npms.io/search?q=ponyfill.
    at isNative (/Users/raphaelchampeimont/nodejs-doctrine/node_modules/lodash/isNative.js:35:11)
    at setupJSON (/Users/raphaelchampeimont/nodejs-doctrine/node_modules/rollbar/src/utility.js:13:9)
    at Object.<anonymous> (/Users/raphaelchampeimont/nodejs-doctrine/node_modules/rollbar/src/utility.js:25:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)`

When downgrading to Rollbar 2.2.3 it works again.

I am currently investigating the bug and trying to produce a minimal example that reproduces the bug.

**Comment by almacha** [comment]
After investigation it seems that the problem appears when loading both modules Rollbar and libphonenumber-js.

Minimalist code to reproduce:
`const libphonenumber = require('libphonenumber-js')
const Rollbar = require('rollbar');
`

My libphonenumber-js version is 0.4.27

**Comment by almacha** [comment]
Here is a minimalist package.json that allows to reproduce the bug with the minimal list I published above:

````
{
  "name": "test",
  "version": "1.0.0",
  "private": true,
  "dependencies": {
    "libphonenumber-js": "^0.4.7",
    "rollbar": "^2.0.2"
  }
}
```

**Comment by rokob** [comment]
I reverted the offending commit and release v2.2.5 so that this is no longer happening until I can fully debug what the issue is.

**Comment by almacha** [comment]
Thank you for your rapid reaction. Indeed the bug is fixed in 2.2.5.
